packages/atomic/cypress/integration/facets/facet-common-actions.ts

```diff
@@ -11,31 +11,103 @@ export function selectIdleCheckboxValueAt(
   FacetWithCheckboxSelector: FacetWithCheckboxSelector,
   index: number
 ) {
```

Comment:
This action is pretty complex. Unfortunately, it kinda needed to be if we want these functions to be self-sufficient (not requiring assertions in `*.cypress.ts` files).